src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -126,12 +112,22 @@ export function AccessCodeTable({
     [setSelectedEditAccessCodeId]
   )
 
+  const handleAccessCodeDeleteSuccess = useCallback((): void => {
+    setAccessCodeResult('deleted')
+  }, [])
```

Comment:
I'm surprised this isn't a linter error for a missing dependency. 
```suggestion
  const handleAccessCodeDeleteSuccess = useCallback((): void => {
    setAccessCodeResult('deleted')
  }, [setAccessCodeResult])
```

Reply:
oops indeed, yes my linter has been hit-or-miss for effect deps for some reason
The CI passed too 🤔